src/CachePeers.h

```diff
@@ -20,6 +20,8 @@
 class CachePeers
```

Comment:
If this code survives the other/primary change requests:

To prevent bugs if somebody decides to derive from this non-virtual destructor class:

```suggestion
class CachePeers final
```

Reply:
Adjusted.